libmscore/score.h

```diff
@@ -1336,16 +1341,16 @@ class MasterScore : public Score
     QFileInfo info;
 
     bool read(XmlReader&);
-    void setPrev(MasterScore* s) { _prev = s; }
-    void setNext(MasterScore* s) { _next = s; }
 
 public:
     MasterScore();
     MasterScore(const MStyle&);
     virtual ~MasterScore();
     MasterScore* clone();
 
-    virtual bool isMaster() const override { return true; }
+    virtual ElementType type() const override { return ElementType::SCORE; } // TODO: if I change it to MasterScore scores are not drawn in album-mode
+
+    virtual bool isMaster() const override { return true; } // TODO: should this be removed?
```

Comment:
This could be replaced by isMasterScore (and probably will be in the future if those classes are rewritten) but as of right now a MasterScore needs to have ElementType::SCORE (otherwise nothing gets drawn on the screenview). So right now toMasterScore/isMasterScore are useless.